docs/apis/cli.md

```diff
@@ -248,6 +252,16 @@ Action "cancel" cancels a running program.
                                    configuration.
 
 
+Action "stop" stops a running program (streaming jobs only).
```

Comment:
Maybe we should add that it a stop signal without strong consistency guarantees.